pyClient/zeth/encryption.py

```diff
@@ -61,13 +60,63 @@ def encryption_public_key_from_hex(pk_str: str) -> EncryptionPublicKey:
 
 def get_encryption_public_key(
         enc_secret: EncryptionSecretKey) -> EncryptionPublicKey:
+    """
+    Derive publickeyf rom secret key
```

Comment:
`Derive publickeyf rom secret key` -> "Derive the public key from the secret key"

Reply:
Changed this comment.